src/builtin_plugins/mathjax/index.js

```diff
@@ -4,18 +4,23 @@ const mjpage = require('mathjax-node-page')
 
 exports.constructor = async function (params) {
   return {
-    htmlModifiers: [ asyncMathjax ]
+    htmlModifiers: [asyncMathjax(params)]
   }
 }
 
-var asyncMathjax = async function (html) {
-  return new Promise(resolve => {
-    mjpage.mjpage(html, {
-      format: ['TeX']
-    }, {
-      mml: true,
-      css: true,
-      html: true
-    }, response => resolve(response))
-  })
-}
+var asyncMathjax =
+  (params) => {
+    console.log(params);
```

Comment:
Should we remove `console.log`?

Reply:
Yes, but it looked like the maintainer wasn't merging PRs anymore so I didn't bother.